modules/wowza/automation.tf

```diff
@@ -0,0 +1,32 @@
+resource "azurerm_automation_account" "vm-start-stop" {
+
+  name                = "${var.product}-recordings-${var.env}-aa"
+  location            = var.location
+  resource_group_name = "${var.product}-recordings-${var.env}-rg"
```

Comment:
can get this from azurerm_resource_group.name

Reply:
Updated.

Reply:
not changed?

Reply:
Updated